### DISCUSSION TITLE
Invalid response from S2 multicast setValue command

### DISCUSSION THREAD

**Discussion by kpine** [zwave-js/zwave-js#6695] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?

An invalid result is returned for a S2 multicast setValue command. The result is the object `{"nodeId":255,"ccId":"Supervision","ccCommand":"0x02"}` but it should be a `SetValueResult` instead.

The invalid result causes exceptions when submitted via HA's Z-Wave integration, as it expects a `SetValueResult` return value. See https://github.com/home-assistant/core/issues/111685.

What did you expect to happen instead?

The method should return `SetValueResult`, example: `{"status":254}`.

I used ZUI driver code to exclude HA as the problem, and could reproduce. Run this driver code, where the multicast targets are included with Security S2:

```javascript
const { logger } = this

const virtualNode = driver.controller.getMulticastGroup([39, 43]);
const result = await virtualNode.setValue(
    {"commandClass": 38, "property": "targetValue"},
    24,
);

logger.info(`Response: ${JSON.stringify(result)}`);
```

You will see a log message like:

```text
2024-03-12 21:48:21.340 INFO Z-WAVE: Response: {"nodeId":255,"ccId":"Supervision","ccCommand":"0x02"}
```

Anything else we should know?

It's not a problem when the multicast targets are included with no security. With the same driver code I see:

```text
2024-03-12 22:39:47.110 INFO Z-WAVE: Response: {"status":254}
```

In the attached log file I used nodes 39 and 43 (S2 Auth) for the invalid response, and 2 and 14 (no security) for the working one.

### Software versions

zwave-js-ui: 9.9.0.09b107a
zwave-js: 12.4.4

### Device information

N/A

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwave_js.log](https://github.com/zwave-js/node-zwave-js/files/14582870/zwave_js.log)

**Comment by AlCalzone** [comment, marked as answer]
Escalated to an issue: https://github.com/zwave-js/node-zwave-js/issues/6696
